src/ext/resources/legacygraphqlapi/schema.graphqls

```diff
@@ -1424,7 +1424,16 @@ type Leg {
         Transportation modes for which all stops in the parent station are used as possible destination stops
         for the next legs. For modes not listed, only the exact destination stop of the leg is considered.
         """
-        destinationModesWithParentStation: [TransitMode!]): [Leg!]
+        destinationModesWithParentStation: [TransitMode!]
+
+        """
+        Whether all possible visits to the destination stops parent station should be treated as a new alternative leg.
+        If true, only the first visit is used to generate an alternative.
+        Otherwise a single trip might yield multiple alternative legs.
+        """
+        onlyFirstDestinationStop: Boolean = true
```

Comment:
Do we ever need to set this to false, do we need the parameter?

Reply:
I don't think there is any need for the parameter at least at the moment but I thought that it might be good to allow potential other users choose what kind of behavior they want from the query. I would be ok with removing the parameter though and only adding it if someone found an actual need for it

Reply:
I'd do that.